src/extension.ts

```diff
@@ -245,11 +260,11 @@ export function activate(context: vscode.ExtensionContext) {
 		if (selected.length > 0) {
 			showMeld([selected, path]);
 		} else {
-			window.showErrorMessage("Meld Diff: First select a file to compare to!")
+			window.showErrorMessage("Meld Diff: First select a file to compare to!");
 		}
 	}));
 }
 
 
 // eslint-disable-next-line @typescript-eslint/no-empty-function
-export function deactivate() {}
+export function deactivate() { }
```

Comment:
Please add the clean up also an `deactivate` so the files are also deleted if vscode is closed before meld.
```ts
export function deactivate(): undefined {
	cleanupTmpFiles(filesToRemove);
	return undefined;
}
```

Reply:
Think this is not really a good idea. The "filesToRemove" aren't within global scope, since the files get tracked per meld instance for good reason. (Commit 2)

Would consider this as an edge case anyway. Not sure if meld child processes would die anyway if vscode gets closed.
But it's your code, you can do whatever you want.

Reply:
Meld child processes will not be closed when vscode is closed.

I see, I have to implement a way to remember the not closed temp files. Maybe it's a good way to ask to user on vscode close if he want to delete the tmp files or not.